admin/functions-notice.php

```diff
@@ -85,4 +85,75 @@ function aioseop_notice_disable_woocommerce_detected_on_nonpro() {
 		global $aioseop_notices;
 		$aioseop_notices->deactivate_notice( 'woocommerce_detected' );
 	}
+
+	/**
+	 * Set Notice on Activation to Review Plugin
+	 *
+	 * A delayed notice that is set during activation, or initialization (old installs),
+	 * to later display a review/rate AIOSEOP plugin. Delay time: 12 days.
+	 * Delay "...give me a week." 5 days
+	 *
+	 * @since 2.4.2
+	 *
+	 * @global AIOSEOP_Notices $aioseop_notices
+	 *
+	 * @param boolean $update Updates the notice with new content and configurations.
+	 * @param boolean $reset  Notice are re-initiated.
+	 */
+	function aioseop_notice_set_activation_review_plugin( $update = false, $reset = false ) {
+		global $aioseop_notices;
+
+		// TODO Optimize - Create a callback function/method to store most of the configurations (Avoid Database concept).
+		// Dynamic variable could be stored in the database. Config functions could go into a config file/folder.
+		$notice = aioseop_notice_review_plugin();
+
+		if ( $aioseop_notices->insert_notice( $notice ) ) {
+			// aioseop_footer_set_review();
+		} elseif ( $update ) {
+			$aioseop_notices->update_notice( $notice );
+
+			if ( $reset ) {
+				$aioseop_notices->activate_notice( $notice['slug'] );
+				// aioseop_footer_remove_review();
+				// aioseop_footer_set_review();
+			}
+		}
+	}
+
+	/**
+	 * Notice - Review Plugin
+	 *
+	 * @return array
```

Comment:
a description of what it returns would help

Reply:
Added